src/main/java/spoon/testing/utils/ModelUtils.java

```diff
@@ -29,36 +29,46 @@
 import java.io.FileNotFoundException;
 
 public final class ModelUtils {
+
+	public static Launcher lastLauncher;
```

Comment:
I need it because I do not know how to do it better...
Have a look at testCtParameterRemoveRefactoring 
```java
	@Test
	public void testCtParameterRemoveRefactoring() {
		String testPackagePath = "spoon/test/refactoring/parameter/testclasses";
		Factory factory = ModelUtils.build(new File("./src/test/java/"+testPackagePath));
		Launcher launcher = ModelUtils.getLastLauncher();
		
		CtType<?> typeA = factory.Class().get(TypeA.class);
...
//I need launcher to print model to java. Is there nicer way?
		launcher.setSourceOutputDirectory(new File("./target/spooned/"));
		launcher.getModelBuilder().generateProcessedSourceFiles(OutputType.CLASSES);
		ModelUtils.canBeBuilt("./target/spooned/"+testPackagePath, 8);

```